### ISSUE TITLE
TcNo Crashing on startup

### ISSUE THREAD

**Issue by Uphorium** [TcNobo/TcNo-Acc-Switcher#282]
I'm getting this error message every time I start TcNo Switcher. I have uninstalled and reinstalled as well as cleared cache and so far not been able to resolve it. I also tried using CEF as the ActiveBrowser but it fails to install after running with administrator. The issue started about the beginning of this month.
![TcNo-Acc-Switcher_main_71igeyJs9E](https://user-images.githubusercontent.com/25294600/209583698-376f39fd-47a5-48ea-9db3-470cc7f8782b.png)

**Comment by Unknown047** [comment]
I'm facing the same issue, could you find a solution of this as of now?

**Comment by Uphorium** [comment]
No, but after all the above steps and reinstalling the redistributables I can now use it again but still get the message. Before I would get the message and then it would auto close.

**Comment by Unknown047** [comment]
The tray application works in my pc though, none other does.

**Comment by TCNOco** [comment]
This issue relates to an HTTP connection failing, likely for anonymous stats, update checks or something else. The ode has changed a lot since it's release, and I'm quite sure this was addressed. Please let me know if you're still experiencing this!

Check the [latest release](https://github.com/TCNOco/TcNo-Acc-Switcher/releases/latest)
Closing this issue as resolved. Thanks for the input.
If there are any issues, let me know on Discord, or through a new issue.
